doc/GETSTARTED.md

```diff
@@ -84,6 +90,71 @@ Optionally you can also run tests to verify the installed Elektra, see [here for
 kdb run_nokdbtests
 ```
 
+## Hello first steps!
+
+This section attempts to give a brief introduction on how to use the key database of Elektra. The key database is modified and queried by using the `kdb` tool.
+
+Let's look into an example:
+
+```sh
+sudo kdb mount "hello.spec.ni" "spec:/example/hello" ni
+```
+
+This first command shows one of the core concepts: In Elektra you mount a file into the key database by specifying a mountpoint.
+In this example we mount the file `hello.spec.ni` that shall be our configuration specification to the mountpoint `/example/hello`
+in the `spec` [namespace](/doc/tutorials/namespaces.md) (which is the [namespace](/doc/tutorials/namespaces.md) of the specification).
+We could also say that we persistently mount a new _backend_. The last parameter `ni` specifies that we use the [ni plugin](/src/plugins/ni/README.md)
+to write in the ni format to the file. This can also be omitted in which case the [dump plugin](/src/plugins/dump/) is used.
+
+Since we did not denote an absolute file path it depends on the namespace where the actual file is stored.
+However, we can always retrieve the location of a file with the `file` subcommand, e.g., `kdb file "spec:/example/hello"`.
+
+Now let's add some data to our specification:
+
+```sh
+sudo kdb set "spec:/example/hello/what" ""
+```
+
+As you see above we extended the the mountpoint `/example/hello` with `/what`. Our specification now requires a `what` key in the configuration.
+Note that the value is empty since we just want to specify the key.
+
+Now let's show the actual power of Elektra by setting metadata to our specification:
+
+```sh
+sudo kdb meta-set "spec:/example/hello/what" default World
+```
+
+The metakey `default` (having the value `World`) is now associated with the key `spec:/example/hello/what`.
+It should be noted that the previous setting of `what` is in this scenario redundant and could be omitted.
```

Comment:
If you mean removing `sudo kdb set "spec:/example/hello/what" ""` then I disagree because this is an fundamental and integral component of the tutorial. Even if it's redundant it enhances the desciption below and shows that there is a difference between set and meta-set. So (from my perspective) I'd find it more confusing if it is removed.

If you mean removing the "It should be noted [...]" sentence then I'm more tempted to extend it with "[...] because this is done implicitly".

Reply:
I was talking about the `kdb set` line. However, if you think it helps with the explanation then keep it in.

Reply:
OK. Like I said I think it helps with the explanation. This might be matter of opinion so there is no right or wrong. Anyhow, thanks for the suggestion. :-)